apps/condo/domains/ticket/tasks/reopenDeferredTicketsTask.js

```diff
@@ -0,0 +1,33 @@
+const { createCronTask } = require('@core/keystone/tasks')
+const { getSchemaCtx, find } = require('@core/keystone/schema')
+const { STATUS_IDS } = require('@condo/domains/ticket/constants/statusTransitions')
+const dayjs = require('dayjs')
+const { Ticket } = require('@condo/domains/ticket/utils/serverSchema')
+
+/**
+ * Opens tickets that are in the "deferred" status and the date they are deferring has expired.
+ * And resets the executor and assignee of this ticket
+ */
+const reopenDeferredTicketsTask = createCronTask('reopenDeferredTickets', '0 0 * * *', async () => {
+    const { keystone } = await getSchemaCtx('Ticket')
+    const adminContext = await keystone.createContext({ skipAccessControl: true })
+    const currentDate = dayjs().startOf('day').toISOString()
+
+    const ticketsToChange = await find('Ticket', {
```

Comment:
there could be thousands of deferred tickets, we have limits on requests... so this could fail silently on background and we wouldn't know anything until someone notices, that deferred tickets aren't reopened anymore.

Reply:
And what to do in this case?
Auto-closing completed tickets works the same way

Reply:
chunks are now used